.travis.yml

```diff
@@ -9,3 +9,4 @@ python:
 # install: "sudo apt-get install"
 # command to run tests, e.g. python setup.py test
 script: python cpplint_unittest.py
+script: python nitpick_unittest.py
```

Comment:
And now I'm answering the email thread via email

On Sun, May 31, 2015 at 11:14 AM, Itamar Ostricher <notifications@github.com
> wrote:

> In .travis.yml
> <https://github.com/yowza3d/cpplint/pull/1#discussion_r31388037>:
>
> > @@ -9,3 +9,4 @@ python:
> >  # install: "sudo apt-get install"
> >  # command to run tests, e.g. python setup.py test
> >  script: python cpplint_unittest.py
> > +script: python nitpick_unittest.py
>
> @eyalyowza <https://github.com/eyalyowza> - this is the breaking line
>
> side note - it was somewhat annoying to get to the diff view for
> line-level comment
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yowza3d/cpplint/pull/1/files#r31388037>.
>